test/util/assertAriaRole.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+
+/**
+ * Confirm the aria-labelledby element.
+ *
+ * @param {obj} t             - ava execution object
+ * @param {String} exampleId  - the id of the element containing the example
+ * @param {String} role       - an aria role
+ * @param {String} roleCount  - the number of elements with role set for example
+ * @param {String} elementTag - the element the role should be found on
+ */
+
+module.exports = async function assertAriaRole (t, exampleId, role, roleCount, elementTag) {
```

Comment:
Maybe rename this to assertAriaRoles since you are checking for (possibly) multiple instances of a role